seed-db/scripts/course.js

```diff
@@ -1,17 +1,52 @@
-const faker = require("faker");
-
+const faker = require('faker');
+const regulations = require('./regulation')
 const generateCourses = (amount) => {
     const courses = [];
     for (let i = 0; i < amount; i++) {
-        courses.push(createCourse());
+        courses.push(createCourse(i));
     }
     return courses;
 };
 
-const createCourse = () => {
+getRandomElement = (array) => {
+    return array[Math.floor(Math.random() * array.length)]
+}
+
+const points = [15]
+const semesters = ["S1", "S2", "SS"]
+const subjects = ["SOFTENG", "COMPSCI", "ENGGEN"]
+
+const generateCourseList = () => {
+    const course0 = []
+    for (let i = 0; i < 10; i++) {
+        course0.push(getRandomElement(subjects) + " " + Math.floor(Math.random() * 999))
+    }
+    return course0
+}
+
+const courses = generateCourseList()
+
+const generateRandomCourseList = () => {
+    const courseList = []
+    for (let i = 0; i < 3; i++) {
+        courseList.push(getRandomElement(courses))
+    }
+    return courseList
+}
+const regulationsArray = regulations.generateRegulations(10)
+const createCourse = (i) => {
     // Make this smarter
     const course = {
-        courseName: faker.name.findName(),
+        _id: "5f4e14016282ae3706ba54f" + i,
```

Comment:
What is this id for lol?

Reply:
regulations which require course _id in the schema

Reply:
For the actual course object, this would be randomised right?

Reply:
yea